src/test/e2e/22_git_and_gitops_test.go

```diff
@@ -17,20 +18,20 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestGitAndFlux(t *testing.T) {
-	t.Log("E2E: Git and flux")
+func TestGit(t *testing.T) {
+	t.Log("E2E: Git")
 	e2e.SetupWithCluster(t)
 	tmpdir := t.TempDir()
 	cachePath := filepath.Join(tmpdir, ".cache-location")
 
-	buildPath := filepath.Join("src", "test", "packages", "22-git-and-flux")
-	stdOut, stdErr, err := e2e.Zarf("package", "create", buildPath, "--zarf-cache", cachePath, "-o=build", "--confirm")
+	buildPath := filepath.Join("src", "test", "packages", "22-git-data")
+	stdOut, stdErr, err := e2e.Zarf("package", "create", buildPath, "--zarf-cache", cachePath, "-o=build", "--confirm", "-a=arm64")
```

Comment:
```suggestion
	stdOut, stdErr, err := e2e.Zarf("package", "create", buildPath, "--zarf-cache", cachePath, "-o=build", "--confirm")
```
The architecture should be removed since the test will use e2e.Arch (and in GitHub that is `amd64`)

Reply:
ah yes, left over from testing, my bad.